src/overlays/actors/ovl_En_Ge3/z_en_ge3.c

```diff
@@ -28,49 +46,386 @@ const ActorInit En_Ge3_InitVars = {
     (ActorFunc)EnGe3_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_809A0DA0 = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0x01000222, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00000000, 0x00, 0x00 },
+        { 0x01000222, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_ON,
+    },
     { 20, 50, 0, { 0, 0, 0 } },
 };
 
-#endif
+void EnGe3_Init(Actor* thisx, PlayState* play) {
+    s32 pad;
+    EnGe3* this = THIS;
+
+    ActorShape_Init(&this->picto.actor.shape, 0.0f, ActorShadow_DrawCircle, 20.0f);
+    SkelAnime_InitFlex(play, &this->skelAnime, &gGerudoRedSkel, NULL, this->jointTable, this->morphTable,
+                       GERUDO_RED_LIMB_MAX);
+    Animation_PlayLoop(&this->skelAnime, &gGerudoRedHandOnHipWaitAnim);
+    Collider_InitAndSetCylinder(play, &this->collider, &this->picto.actor, &sCylinderInit);
+    this->picto.actor.colChkInfo.mass = MASS_IMMOVABLE;
+    Actor_SetScale(&this->picto.actor, 0.01f);
+
+    this->animIndex = GERUDO_AVEIL_ANIM_NONE;
+    this->csAction = GERUDO_AVEIL_CSACTION_NONE;
+    this->picto.validationFunc = EnGe3_ValidatePictograph;
+    EnGe3_SetupPath(this, play);
+
+    if (GERUDO_AVEIL_GET_TYPE(&this->picto.actor) == GERUDO_AVEIL_TYPE_AVEILS_CHAMBER) {
+        EnGe3_ChangeAnim(this, GERUDO_AVEIL_ANIM_HAND_ON_HIP_WAIT, ANIMMODE_LOOP, 0.0f);
+        this->actionFunc = EnGe3_AveilsChamberIdle;
+
+        if (gSaveContext.save.weekEventReg[83] & 2) { // Knocked beehive down
+            Actor_MarkForDeath(&this->picto.actor);
+            return;
+        }
+    } else {
+        EnGe3_ChangeAnim(this, GERUDO_AVEIL_ANIM_WALK, ANIMMODE_LOOP, 0.0f);
+        this->actionFunc = EnGe3_Idle;
+        this->picto.actor.speedXZ = 1.5f;
+    }
+
+    this->stateFlags = 0;
+    this->picto.actor.targetMode = 6;
+    this->picto.actor.terminalVelocity = -4.0f;
+    this->picto.actor.gravity = -1.0f;
+    gSaveContext.save.weekEventReg[80] &= (u8)~8;
+}
+
+void EnGe3_Destroy(Actor* thisx, PlayState* play) {
+    EnGe3* this = THIS;
+
+    Collider_DestroyCylinder(play, &this->collider);
+}
+
+void EnGe3_ChangeAnim(EnGe3* this, s16 index, u8 mode, f32 morphFrames) {
+    static AnimationHeader* sAnimations[] = {
+        &gGerudoRedStandAnim,         /* GERUDO_AVEIL_ANIM_STAND */
+        &gGerudoRedWalkingAnim,       /* GERUDO_AVEIL_ANIM_WALK */
+        &gGerudoRedHandOnHipWaitAnim, /* GERUDO_AVEIL_ANIM_HAND_ON_HIP_WAIT */
```

Comment:
```suggestion
        &gGerudoRedStandAnim,         // GERUDO_AVEIL_ANIM_STAND
        &gGerudoRedWalkingAnim,       // GERUDO_AVEIL_ANIM_WALK
        &gGerudoRedHandOnHipWaitAnim, // GERUDO_AVEIL_ANIM_HAND_ON_HIP_WAIT
```
Is the style we used I believe